admin_ui/src/main.ts

```diff
@@ -45,6 +47,7 @@ Vue.filter("readable", function (value) {
 /*****************************************************************************/
 
 Vue.config.productionTip = false
+Vue.use(Chartkick.use(Chart))
 
```

Comment:
```suggestion
// charts font color and gridlines in dark mode
if (localStorage.getItem('darkMode')) {
    Chart.defaults.global.defaultFontColor = "#ffffff"
    Chart.defaults.scale.gridLines.color = "#ffffff"
}
```

Reply:
This is a way to change Chart.js colors in dark mode, but unfortunately it doesn't work without refreshing the page. I'm sorry I didn't check before posting the suggested change.

Reply:
No worries, it's still useful - there might be a way to reload the chart without reloading the entire page.